src/blister.h

```diff
@@ -209,29 +209,88 @@ static inline UInt * BLOCKS_BLIST(Obj list)
 /****************************************************************************
 **
 *F  COUNT_TRUES_BLOCK( <block> )  . . . . . . . . . . . count number of trues
+*F  COUNT_TRUES_BLOCKS( <ptr>, <nblocks> ) 
+**
+**  This function uses either the gcc builtin __builtin_popcount which
```

Comment:
"This function" now is ambiguous.  In other comments, we write something like this:
> 'COUNT_TRUES_BLOCK' uses either ...

Reply:
Alternatively, move `COUNT_TRUES_BLOCKS` to its own documentation comment